infra/Pos/Communication/Relay/Logic.hs

```diff
@@ -310,10 +341,11 @@ invDataListener enqueue InvReqDataParams{..} = listenerConv $ \__ourVerInfo node
             whenJust inv' $ expectInv $ \InvMsg{..} -> do
                 useful <- handleInvDo (handleInv nodeId) imKey
                 whenJust useful $ \ne -> do
```

Comment:
Hmm. This brings up a similar point that there is no acknowledgement in the not useful case, it doesn't say no it just stops, which is not distinguishable from failure.